app/actions.js

```diff
@@ -343,7 +381,7 @@ export const viewChannel = ({ addr, channel, skipScreenHistory }) => (dispatch,
     client.focusChannel(channel)
     client.markChannelRead(channel)
   } else {
-    dispatch(joinChannel({ addr, channel }))
+    // dispatch(joinChannel({ addr, channel }))
```

Comment:
```suggestion
```

Reply:
This is the main issue that came up with the cabal-client update which threw the app into a loop. It seems that joinChannel isnt really needed here though as things seem to work as expected without it. My guess is other bugs in Desktop were hiding the real problem.

Reply:
maybe we can encode this in a comment above the line? would be nice to keep the context intact, in case there's a bug in cabal-client somehow :)

Reply:
Ok!